cli/generate.go

```diff
@@ -73,13 +73,17 @@ var generateCmd = cli.Command{
 			Name:  "exclude,x",
 			Usage: "Files or directories to exclude from this run, relative to the chart.",
 		},
+		cli.BoolFlag{
+			Name:  "force,f",
+			Usage: "Forces to run generate subcommands",
```

Comment:
can you be a little more clear here by indicating that generate commands will be run regardless if they've already been run. For example:

"Forces generate commands to run, even if they have before. Overwrites generated templates if generation had already run"